### PR TITLE
Add support for ws:// scheme for libSQL

### PR DESCRIPTION
**Pull request by george-lim**
Addresses #1951

### CODE REVIEW

File: doc/md/guides/sqlite/turso.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ slug: /guides/sqlite/turso
 ---
 [Turso](https://turso.tech) is an edge-hosted, distributed database based on
 [libSQL](https://github.com/libsql/libsql), an open-source and open-contribution
-fork of SQLite. It was designed to minimize query latency for applications where 
-queries come from anywhere in the world. 
+fork of SQLite. It was designed to minimize query latency for applications where
+queries come from anywhere in the world.
```
Comment (a8m) [inline doc/md/guides/sqlite/turso.md:9]: Please revert the irrelevant changes

File: doc/md/guides/sqlite/turso.md [hunk 2]
```diff
@@ -28,8 +28,8 @@ your Turso databases.
    ```
    turso auth signup
    ```
-   The CLI launches your default browser and asks to log in with GitHub. 
-   The first time you log in, you are asked to grant the GitHub Turso 
+   The CLI launches your default browser and asks to log in with GitHub.
```
Comment (a8m) [inline doc/md/guides/sqlite/turso.md:31]: Ditto

**Review by a8m — APPROVED** [review]

**Comment by a8m** [comment]
Thanks for the contribution, @george-lim.

**Comment by george-lim** [comment]
no worries, thanks a lot @a8m :)

**Comment by zestsystem** [comment]
Can http:// scheme be supported as well? ws:// scheme to connect to sqld does not work when using in a serverless environment such as Cloudflare Worker, Vercel Edge, and Netlify?

**Pull request merged**